bin/startup

```diff
@@ -12,4 +16,4 @@ fi
 echo "starting rails"
 rm -f tmp/pids/server.pid
 
-bundle exec rails s -b '0.0.0.0'
+exec bundle exec rails s -b '0.0.0.0'
```

Comment:
`_exec_ bundle exec`?  Seems like a lot of "execs".  Doesn't this run in the container?